app/posts/digital-public-works-ideas.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Ideas for digital public works
+description: Some thoughts on digital platforms and services that would be helpful for cross-government teams.
+date: 2024-03-01
+author:
+  name: Frankie Roberto
+  url: https://github.com/frankieroberto
+---
+
+A recent blog post from dxw suggests that [there needs to be more support for digital public works](https://www.dxw.com/2024/01/there-needs-to-be-more-support-for-digital-public-works/), described as platforms and tools which work across government to make it easier and faster to build services.
+
+I’m also big supporter of these. This unofficial X-GOVUK community is one way to support cross-government collaboration and tools, but many digital public works would need full time teams to support them.
+
+There’s lots to be thankful for. The [platforms developed by the Government Digital Service](https://www.gov.uk/service-toolkit#platforms-and-tools) – Notify, Pay, One Login and Forms – are all super useful.
+
+Unfortunately there doesn’t currently seem to be an easy way for government departments working on services to spot other opportunities for common platforms and to turn these into funded cross-government teams and services.
+
+However, were this to be possible, I do have a list of ideas for what these services could be.
+
+## The ideas
+
+### 1. Analytics
+
+There are plenty of existing commercial and open source tools for this, so I’m not suggesting rebuilding those, but instead picking one and building a service on top of it. 
+
+The team would do the hard work of navigating the trade-offs between privacy, security and usability to give teams the design patterns and technical implementation for collecting consent and then data.
+
+They might also build a simple web service to help teams view and understand the most important metrics, with comparisons across services.
```

Comment:
Getting déjà vu of the performance platform here!

Reply:
But, soft! what light through yonder window breaks?

Reply:
😂